LEAF_Nexus/templates/view_group.tpl

```diff
@@ -343,24 +344,88 @@ function confirmUnlinkEmployee(empUID) {
 }
 
 function confirmDeleteTag(inTag) {
-    var warning = '';
-    warning = '<br /><br /><span style="color: red">WARNING!! removal of service would potentially impact your org chart structure, if you are trying to grant service chief access go to Request Portal->Admin panel-> Service Chief</span>';
-
-    confirm_dialog.setContent('<img src="dynicons/?img=help-browser.svg&amp;w=48" alt="" style="float: left; padding-right: 16px" /> <span style="font-size: 150%">Are you sure you want to delete this tag?</span>'+ warning);
-    confirm_dialog.setTitle('Confirmation');
-    confirm_dialog.setSaveHandler(function() {
-        $.ajax({
-        	type: 'DELETE',
-            url: './api/group/<!--{$groupID}-->/tag?' +
-                $.param({tag: inTag,
-                         CSRFToken: '<!--{$CSRFToken}-->'}),
-            success: function(response) {
-                window.location.reload();
-            },
-            cache: false
-        });
+    let validate = {'groupID': '<!--{$groupID}-->'};
+    $.ajax({
+        type: 'GET',
+        url: './api/platform/portal/_<!--{$orgchart_path}-->',
+        success: function(response) {
+            if (response.constructor === Array) {
+                response.forEach(function (item, index) {
+                    item.tags.forEach(function (tag) {
+                        if (tag === inTag) {
+                            // need to check the portal for this tag in this group
+                            $.ajax({
+                                type: 'GET',
+                                url: '..' + item.site_path + '/api/group/list',
+                                success: function (response) {
+                                    console.log(response);
+                                    let found = false;
+
+                                    response.forEach(function (group) {
+                                        if (group.groupID === Number(validate.groupID)) {
+                                            found = true;
```

Comment:
If a normal loop is used here instead of a forEach you should be able to use a break in it.

Array method 'some' returns a bool and would also work here
found = reponse.some(group => group.groupID === Number(validate.groupID))